### ISSUE TITLE
Do you ever get so absorbed in coding that you neglect "life" stuff?

### ISSUE THREAD

**Issue by Azuriru** [mxcl/AmA#29]
And do you tend to enjoy the feeling? And how long does this codeaholic usually last? Sorry for weird question haha

**Comment by mxcl** [comment]
Yes totally, it's happened many times. Homebrew was a good example. I was single and while I was writing it I just ate (sporadically), worked, slept (occasionally) and ignored life (friends, family, bills, etc.).

Also while working on Amarok I ignored everything until my parents kicked me out the house.

It cannot last forever since eventually some part of life, be it people or responsibilities pounce. In my experience though the “holes” are productive and useful and the consequences not severe (provided you don't have kids or other dependents!).

**Comment by Azuriru** [comment]
Yeah I agree it makes you super productive but I feel like it's slightly detrimental to the friendgroups that actively depend on my presence. Thanks for sharing your perspective

**Comment by mxcl** [comment]
If you have people that depend on you then they have to come first.
